README.md

```diff
@@ -57,9 +57,48 @@ This driver supports only one topology key:
 that represents availability by zone.
 
 ## Kubernetes User Guide
-
 ### Install Driver
 
+#### Preflight (Ubuntu like)
+1. You need to install golang
+```
+sudo apt update && sudo apt install golang
+```
+2. Set path for go
+You can read [here](https://github.com/golang/go/wiki/SettingGOPATH) more or quick run this steps
+Set path to /home/$USER/go
+```
+export GOPATH=$HOME/go
+```
+3. Clone the repo to path below
+```
+mkdir -p $GOPATH/src/sigs.k8s.io
+cd $GOPATH/src/sigs.k8s.io
+git clone https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver.git
```

Comment:
wanted to avoid giving users a specific way to clone the repo since they may want to use SSH vs. HTTPS.

I have proposed an alternate version here. Would appreciate if you could comment and review